internal/manager/testdata/lb-ex-2-exp.cfg

```diff
@@ -26,11 +26,11 @@ frontend stats
   stats refresh 10s
   http-request use-service prometheus-exporter if { path /metrics }
 
-backend ssh-service
-  server c0a80101-0000-0000-0000-000000000001 1.2.3.4:2222 check port 2222
```

Comment:
I guess these should look like gid's now

Reply:
since this is an identifier that would show up in logs, the gid is better to use so that someone running this won't have to guess which origin in a pool has what name